src/middlewares/auth.middleware.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-console */
+import jwt from 'jsonwebtoken';
+
+export const authmiddleware = (req, res, next) => {
+  const auth = req.headers.authorisation;
+
+  if (!auth) {
+    res.sendStatus(401);
+
+    return;
+  }
+
+  const [, token] = auth.split(' ');
+
+  if (!token) {
+    res.sendStatus(401);
+
+    return;
+  }
+
+  try {
+    const data = jwt.verify(token, process.env.JWT_SECRET);
+
+    console.log(data);
```

Comment:
Using `console.log` for logging sensitive data like JWT payloads can be a security risk. Consider using a proper logging mechanism and avoid logging sensitive information.